native/packages/react-native-bpk-component-horizontal-nav/src/BpkHorizontalNav.js

```diff
@@ -42,39 +43,90 @@ const styles = StyleSheet.create({
     flexGrow: 1,
     justifyContent: 'space-around',
   },
+  selectedIndicator: {
+    backgroundColor: tokens.colorBlue700,
+    height: tokens.borderSizeLg,
+    width: 50,
+  },
 });
 
-const BpkHorizontalNav = (props) => {
-  const {
-    children,
-    spaceAround,
-    style,
-    ...rest
-  } = props;
-  const navStyle = [styles.horizontalNav];
-  if (spaceAround) {
-    navStyle.push(styles.spaceAround);
+class BpkHorizontalNav extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isReady: false };
```

Comment:
this is my poor choice of names, I think there might be a better name for this but I can't think of one